cloudfront.tf

```diff
@@ -0,0 +1,96 @@
+resource "aws_cloudfront_distribution" "toolbox_distribution" {
+  depends_on = [
+    aws_acm_certificate.burendo_toolbox
+  ]
+  origin {
+    domain_name              = aws_s3_bucket.burendo_toolbox.bucket_regional_domain_name
+    origin_access_control_id = aws_cloudfront_origin_access_control.toolbox_distribution_acl.id
+    origin_id                = local.s3_origin_id
+    dynamic "custom_header" {
+      for_each = local.custom_origin_headers
+      content {
+        name  = custom_header.value.name
+        value = custom_header.value.value
+      }
+    }
+  }
+
+  enabled             = true
+  is_ipv6_enabled     = true
+  default_root_object = "index.html"
+
+  aliases = local.deploy_fqdn[local.environment] == true ? [local.environment_domain[local.environment]] : []
+
+  default_cache_behavior {
```

Comment:
Add security headers policy here: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-response-headers-policies.html#managed-response-headers-policies-security

Adds more protective headers to the response :)